src/store/login/index.ts

```diff
@@ -16,6 +16,7 @@ export type LoginState = {
 export enum LoginStage {
   EmailLogin = 'email',
   Web3Login = 'web3',
+  ResetPassword = 'reset-password',
```

Comment:
I think we're going to run into a fair bit of confusion between requesting the password reset email and actually resetting the password. Maybe "RequestPasswordReset" for these various things? (can be handled in follow ups)

Reply:
Yeah agreed this makes more sense! Updated :)